content/sensu-enterprise/2.6/guides/using-snow.md

```diff
@@ -0,0 +1,158 @@
+
+This is a quick way to send JSON data to the local Sensu client socket to generate a mock event. If successful, you should see the following in your logs, and inside of SNOW:
+
+{{< highlight json >}}"timestamp":"2018-07-31T01:44:48.936181+0000","level":"info","message":"processing event","event":{"id":"b971c1ca-79a6-44fc-a5b5-0bc2b202c4a0","client":{"name":"sensu-client-01","address":"172.30.0.10","subscriptions":["dev","client:sensu-client-01"],"keepalive":{"handlers":["ec2"],"thresholds":{"warning":30,"critical":60}},"ec2":{"instance_id":"i-0b333990eabb06e35","allowed_instance_states":["running","rebooting"]},"registration":{"handlers":["servicenow"]},"deregister":true,"deregistration":{"handlers":["servicenow"]},"servicenow":{"configuration_item":{"name":"sensu-client-01","cost":"10","asset_tag":"SENSUCLIENT001","assigned_to":"abel.tuter@example.com","department":"development","display_name":"Sensu Test Client"}},"version":"1.4.2","timestamp":1533001478},"check":{"name":"sensu-client-01","output":"THIS IS AN INCIDENT","refresh":10,"status":0,"handlers":["servicenow"],"executed":1533001488,"issued":1533001488,"type":"standard","history":["2","2","2","2","2","2","2","0","2","2","2","2","2","2","2","2","0","2","0","2","0"],"total_state_change":38},"occurrences":1,"occurrences_watermark":1,"last_ok":1533001488,"action":"resolve","timestamp":1533001488,"last_state_change":1533001488,"silenced":false,"silenced_by":[]}}
+
+{"timestamp":"2018-07-31T01:44:49.187061+0000","level":"info","message":"servicenow incident created successfully","short_description":"sensu-client-01/sensu-client-01 : THIS IS AN INCIDENT","event_id":"b971c1ca-79a6-44fc-a5b5-0bc2b202c4a0","contact_name":"default"}{"timestamp":"2018-07-31T01:44:51.021406+0000","level":"info","message":"servicenow incident resolved successfully","short_description":"sensu-client-01/sensu-client-01 : THIS IS AN INCIDENT","event_id":"b971c1ca-79a6-44fc-a5b5-0bc2b202c4a0","contact_name":"default"}{{< /highlight >}}
+
+![snow_incident_fake][5]
+
+## Wrapping It Up
+
+That's all for this guide on configuring and using the Sensu Enterprise SNOW integration. We covered the following topics:
+
+- Configuring the SNOW integration
+- Configuring the client for use with SNOW
+- Configuring checks to use SNOW for incident creation and handling
+
+We hope you've found this useful. For additional resources about the SNOW integration or client SNOW attributes, see the reference links below.
+
+## References
+
+- [SNOW Integration Reference][7]
+- [SNOW Client Attributes][8]
+
+<!-- LINKS -->
+[1]: https://developer.servicenow.com
+[2]: https://slack.sensu.io
+[3]: /images/snow_inventory.png
+[4]: https://github.com/sensu-plugins/sensu-plugins-disk-checks
+[5]: /images/snow_incident.png
+[6]: /images/snow_incident_detail.png
+[7]: ../../integrations/servicenow/
```

Comment:
This link goes to a relative path, but the last link in `[8]` goes to latest. Is there a preferred way to do this site wide?

Reply:
Bumped this to 1.3